phpunit/phpunit/4.7/phpunit.xml.dist

```diff
@@ -10,6 +10,10 @@
     <php>
         <ini name="error_reporting" value="-1" />
         <server name="KERNEL_CLASS" value="App\Kernel" />
+        <server name="APP_ENV" value="test" />
+        <server name="APP_DEBUG" value="0" />
```

Comment:
Both [`APP_ENV` and `APP_DEBUG` vars have no effect][1] on kernel initialization, so new feature or bugfix? ... or simply we can remove them? 

Also note that the current default value for debug mode is `true` in test env, here we are changing this to `false`.

  [1]: https://github.com/symfony/symfony/blob/7ab8efb74ecd422ed627753971637bd7d8dc92ca/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L192-L193

Reply:
Changed false to true for `APP_DEBUG`

Reply:
@yceruto I think we need to add support for these env vars. Can you submit a PR for Symfony 3.4 to use env vars when options are not defined explicitly, and keep the fallback as is if the env vars are not define?

Reply:
Sure, I'm on it.

Reply:
Great! Thank you

Reply:
See https://github.com/symfony/symfony/pull/24177